### DISCUSSION TITLE
Separate shortwave broadcast bands selectable with third button

### DISCUSSION THREAD

**Discussion by cazian** [PE5PVB/TEF6686_ESP32#339] [Ideas]
Would it be possible for the existing shortwave range, which is currently just one single lump from 1700 to 27000kHz, to be broken up into the separate SW broadcast bands, with each press of the third button? With each section starting with the lowest frequency in each official broadcast band (apart from the first step at 1700kHz of course). So, each step would start with the following:
1700kHz
2300
3900
4750
5900
7200
9400
11600
13570
15100
17480
18900
21450
25670
Perhaps make some small downward adjustments on some of these, to allow for common out of band stations?
If that's thought to be too many steps, perhaps break it into larger arbitrary lumps, like 1700, 5000, 10000, 15000, 20000, 25000kHz?
Either would make it easier to navigate through the shortwave range.
Suggestion originally made in Discord group discussion: https://discord.com/channels/1053804249651359765/1053804250276298895/1164978040284119151

**Comment by Overland-DX** [comment]
I don't quite know if I understood you correctly, but we already have an opportunity to choose a band in the SW band.
I think the solution we already have is a very good one.




https://github.com/PE5PVB/TEF6686_ESP32/assets/102610897/d2ea5dad-71e9-4a1a-b2d3-e7e4c4ac457d

**Reply by cazian** [reply]
Oh, thank you, Overland-DX. I had not spotted that setting in the menu in the latest versions, that's a very neat solution and does everything I was after. I will see if I can close this discussion.

**Reply by Overland-DX** [reply]
Glad to be able to help :)
A manual will probably be created when this project is nearing completion.
